### ISSUE TITLE
Test clean_asset_readme action

### ISSUE THREAD

**Issue by alesanchezr** [4GeeksAcademy/About-4Geeks-Academy#3419]
This function [clean_asset_readme](https://github.com/breatheco-de/apiv2/blob/main/breathecode/registry/actions.py#L449) its supposed to process (clean) an asset.readme_raw content.

1. The readme file comes in from github and the content is saved on asset.readme_raw
2. There is [a signal](https://github.com/breatheco-de/apiv2/blob/main/breathecode/registry/signals.py#L8) that triggers every time the asset.readme_raw is modified.
3. The [signal receiver](https://github.com/breatheco-de/apiv2/blob/main/breathecode/registry/receivers.py#L30) triggers a task to clean the content and save the results into asset.readme

Basically the asset.readme_raw content gets cleaned and saved into asset.readme:

```txt
asset.readme_raw => clean_asset_readme => asset.readme
```

## Instructions

We need to test that the cleanup operation its working. The cleanup operation right now consists of:

- [ ] Removing the content in between `<!-- hide -->` and `<!-- endhide -->` blocks of code.
- [ ] Making changes to the relative paths
